app/src/main/java/at/roteskreuz/stopcorona/screens/debug/diagnosis_keys/DebugDiagnosisKeysViewModel.kt

```diff
@@ -33,33 +31,26 @@ class DebugDiagnosisKeysViewModel(
     private val apiInteractor: ApiInteractor,
     private val contextInteractor: ContextInteractor,
     private val exposureNotificationRepository: ExposureNotificationRepository,
-    private val contentDeliveryNetworkDescription: ContentDeliveryNetworkDescription,
+    private val exposureNotificationClient: ExposureNotificationClient,
```

Comment:
@falkorichter If you are not using exposureNotificationRepository for all the communication with the framework, you won't test it properly. In this debug screen something is written, but it's not using the real implementation which needs to be tested.

Consider refactoring.